components/example-notebook-servers/jupyter-pytorch-gaudi/Dockerfile

```diff
@@ -0,0 +1,143 @@
+#
+# NOTE: Use the Makefiles to build this image correctly.
+#
+
+ARG BASE_IMG=<jupyter>
+FROM $BASE_IMG
+
+# Content below is extracted from scripts/Dockerfiles here:
+# https://github.com/HabanaAI/Setup_and_Install/tree/main/dockerfiles/base
+# https://github.com/HabanaAI/Setup_and_Install/tree/main/dockerfiles/pytorch
+
+# version details
+ARG ARTIFACTORY_URL=vault.habana.ai
+ARG VERSION=1.16.2
+ARG REVISION=2
+ARG PYTORCH_VERSION=2.2.2
+ARG OS_NUMBER=2204
+
+# runtime variables
+ENV DEBIAN_FRONTEND=noninteractive
+ENV GC_KERNEL_PATH=/usr/lib/habanalabs/libtpc_kernels.so
+ENV HABANA_LOGS=/var/log/habana_logs/
+ENV HABANA_SCAL_BIN_PATH=/opt/habanalabs/engines_fw
+ENV HABANA_PLUGINS_LIB_PATH=/opt/habanalabs/habana_plugins
+ENV TCMALLOC_LARGE_ALLOC_REPORT_THRESHOLD=7516192768
```

Comment:
These should probably be under a `# configs - habana` section.

Also, feel like `TCMALLOC_LARGE_ALLOC_REPORT_THRESHOLD` needs an explanation of what it's for and why it's necessary in that comment.

Reply:
Top four variables are actually set in habanalab.sh which is sourced, so I'll drop them from the Dockerfile.
The TCMALLOC env variable is obsolete and I'll remove it.